### PR TITLE
docs(confluent): add details for actions pod for confluent

### PR DESCRIPTION
**Pull request by RyanHolstien**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/deploy/confluent-cloud.md [hunk 1]
```diff
@@ -143,6 +143,39 @@ springKafkaConfigurationOverrides:
...
+        consumer_config:
+          security.protocol: ${KAFKA_PROPERTIES_SECURITY_PROTOCOL:-PLAINTEXT}
+          sasl.mechanism: ${KAFKA_PROPERTIES_SASL_MECHANISM:-PLAIN}
+          sasl.username: ${KAFKA_PROPERTIES_SASL_USERNAME}
+          sasl.password: ${KAFKA_PROPERTIES_SASL_PASSWORD}
+```
+
+Specifically `sasl.username` and `sasl.password` are the differences from the base `executor.yaml` example file.
+
+Additionally, you will need to set up secrets for `KAFKA_PROPERTIES_SASL_USERNAME` and `KAFKA_PROPERTIES_SASL_PASSWORD`
```
Comment (jjoyce0510) [inline docs/deploy/confluent-cloud.md:164]: So is the below relevant on helm? If yes lets add that it's only relevant for helm (not docker compose)

Comment (RyanHolstien) [reply docs/deploy/confluent-cloud.md:164]: The settings are relevant for both, but yes this specific way of setting things up is for helm. For docker-compose they'll have to set up the env vars, I can add that.

**Review by jjoyce0510 — COMMENTED** [review]
Let's wait to merge this until we figure out what is going on here..

**Comment by RyanHolstien** [comment]
> Let's wait to merge this until we figure out what is going on here..

Agreed, seems like there are more issues than just the config instructions. If it's going to be in the guide it should work.

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**